eng/vs_tools_check.cmd

```diff
@@ -0,0 +1,40 @@
+@if not defined _echo @echo off
```

Comment:
Would it be better to add an argument to the existing string instead of duplicating the whole existing script except the one line that calls VsDevCmd.bat ?

Reply:
Sorry I hadn't seen this before Jan. Yes, I think this is a good idea to try. I also wasn't very comfortable with duplicating the code but wasn't able to think of another approach. Will try this.